main.c

```diff
@@ -142,20 +144,27 @@ void output(struct window_props* wlist, int n, Window active_window, char* execu
             printf("%%{F%s}", config.active_window_fg_color);
         }
 
+        char* display;
```

Comment:
I don't love this name because my immediate connotation was Xlib `Display`, I would suggest changing this to `window_name` (unless you wanna suggest something else)